src/current/v23.2/cluster-settings.md

```diff
@@ -29,14 +36,42 @@ Use the [`SHOW CLUSTER SETTING`]({% link {{ page.version.version }}/show-cluster
 
 ## Change a cluster setting
 
-Use the [`SET CLUSTER SETTING`]({% link {{ page.version.version }}/set-cluster-setting.md %}) statement.
+Cluster settings can be updated anytime after a cluster has been started. Use the [`SET CLUSTER SETTING`]({% link {{ page.version.version }}/set-cluster-setting.md %}) statement.
 
 Before changing a cluster setting, note the following:
 
 - 	Changing a cluster setting is not instantaneous, as the change must be propagated to other nodes in the cluster.
 
 - 	Do not change cluster settings while [upgrading to a new version of CockroachDB]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}). Wait until all nodes have been upgraded before you make the change.
 
+## Sensitive settings
+
+{% include_cached new-in.html version="v23.2.1" %} Due to security concerns, some cluster settings are considered sensitive. Their values should not be visible to all users when using the [`SHOW CLUSTER SETTING`]({% link {{ page.version.version }}/show-cluster-setting.md %}) statement.
+
+The values of these sensitive settings are always visible to users with the `admin` role or the `MODIFYCLUSTERSETTING` privilege.
+
+The values of these sensitive settings are also visible to users with the `VIEWCLUSTERSETTING` privilege when [`server.redact_sensitive_settings.enabled`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-server-redact-sensitive-settings-enabled) is disabled.
+
+To enable redaction of these sensitive setting values for non-`admin` users without the `MODIFYCLUSTERSETTING` privilege, such as users with the `VIEWCLUSTERSETTING` privilege, run:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING server.redact_sensitive_settings.enabled = 'true';
+~~~
+
+The table below summarizes when sensitive settings values are visible or redacted:
+
+| User attribute | Redaction disabled | Redaction enabled |
```

Comment:
Consider adding a 4th row like

`None of the above roles/privileges | not visible | not visible1